NativeImport/ExampleNativeImportFeed.xml

```diff
@@ -54,6 +54,7 @@
                 <Rating>5</Rating>
                 <IpAddress>64.94.228.137</IpAddress>
                 <Recommended>true</Recommended>
+                <UserEmailAddress>email@address.com</UserEmailAddress>
```

Comment:
Change the domain to example.com. Any domain other than example.com is not allowed.